corehq/apps/linked_domain/local_accessors.py

```diff
@@ -78,3 +79,35 @@ def get_data_dictionary(domain):
 
         data_dictionary[case_type.name] = entry
     return data_dictionary
+
+
+def get_dialer_settings(domain):
+    settings = DialerSettings.objects.get(domain=domain)
```

Comment:
What happens if you try to update these settings but the upstream domain doesn't have a `DialerSettings` object yet?

Reply:
fixed this issue (and with the other integration spaces) and they passed QA